File: CPP_Programming/Day_5/program33.cpp
/*  
    Description : Write a program that accepts frequency & prints
                  the below pattern : 
    Input : 4 #
    Output : #  # # #
*/

#include<iostream>
using namespace std;

class Pattern
{
    public :
        int iNo;
        char ch;

    Pattern(int iNo , char ch)
    {
        this->iNo = iNo;
        this->ch = ch;
    }

    void Display()
    {
        int iCnt = 0;
        
        for(iCnt = 1  ; iCnt <= iNo ; iCnt++)
        {
            cout<<ch<<"\t";
        }

    }

};

int main()
{
    int iValue = 0;
    char ch = '\0';

    cout<<"Enter the frequency : ";
    cin>>iValue;

    cout<<"Enter the character that you want to print : ";
    cin>>ch;

    Pattern pobj(iValue , ch);

    pobj.Display();


    return 0;
}

File: Java_Programming/Day_5/program33.java
/*  
    Description : Write a program that accepts frequency & prints
                  the below pattern : 
    Input : 4 #
    Output : #  # # #
*/

import java.util.*;

class Pattern
{
    public int iNo;
    public String ch;

    Pattern(int iNo , String ch)
    {
        this.iNo = iNo;
        this.ch = ch;
    }

    public void Display()
    {
        int iCnt = 0;

        for(iCnt = 1 ; iCnt <= iNo ; iCnt++)
        {
            System.out.print(ch + "\t");
        }
    }

}

class program33
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter the frequency : ");
        int iValue = sobj.nextInt();

        sobj.nextLine();

        System.out.print("Enter the character that you want to print : ");
        String ch = sobj.nextLine();

        Pattern pobj = new Pattern(iValue , ch);

        pobj.Display();

        sobj.close();
    }
}